docs/library/cohesity_clone_vm.md

```diff
@@ -0,0 +1,118 @@
+# Cohesity Clone Management 
+
+[Go back to Documentation home page ](../README.md)
+
+## Table of Contents
+- [Synopsis](#synopsis)
+- [Requirements](#requirements)
+- [Syntax](#syntax)
+- [Examples](#examples)
+  - [Create a VMware VM clone](#Create-a-VMware-VM-clone)
+  - [Destroy a VMware VM clone](#Destroy-a-VMware-VM-clone)
+- [Parameters](#parameters)
+- [Outputs](#outputs)
+
+## Synopsis
+[top](#cohesity-clone-management)
+
+The Ansible Module is used to clone VMs using snapshots of backup made from VMs.
+
+### Requirements
+[top](#cohesity-clone-management)
+
+* Cohesity DataPlatform running version 6.3 or higher
+* Ansible version 2.6 or higher
+  * The [Ansible Control Machine](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#control-machine-requirements) must be a system running one of the following UNIX operating systems: Linux (Red Hat, Debian, CentOS), macOS, or any of the BSDs. Windows is not supported for the Control Machine.
+* Python version 2.6 or higher
+
```

Comment:
add cohesity management sdk as one of the requirements, add this to clone vm, policy, view modules